anjumol/react_new/section_11/src/components/Modal.jsx

```diff
@@ -0,0 +1,22 @@
+import { useRef, useEffect } from "react";
+import { createPortal } from "react-dom";
+
+function Modal({ open, children, onClose }) {
+  const dialog = useRef();
```

Comment:
change the variable name dialog to dialogRef also change to aother places as well in this file